tests/Integration/Behaviour/Features/Scenario/Customer/customer_private_note.feature

```diff
@@ -0,0 +1,12 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s customer
+@reset-database-before-feature
```

Comment:
I'm not sure we should write 1 dedicated file for it 🤔 are we not going to create _a lot_ of folders then ?

I understand you want to split the scenarios, but if a file contains only 1 scenario, it looks like too much ?

Reply:
Yeah, I agree, but these are "feature" files for a reason, we can't put everything into customer management (which is basically CRUD feature) or some other feature file, do we only have 1 feature about Customers?

How can we improve that? :thinking:

Reply:
Mmmm

All right ☺️ 

Even if later we find we have written too many files, we can merge them because tests do not fall under the SemVer "no BC !" rule

Reply:
Exactly. :+1: